web/themes/new_weather_theme/assets/js/radar.js

```diff
@@ -0,0 +1,116 @@
+  layer: {radar product options by view} // the selected product, default to Super Resolution Base Reflectivity
+  filter: 'WSR-88D', 'TDWR', undefined (all) // type of radar stations to display, default to undefined
+  station: {any station id}, undefined // The selected station, default undefined
+}
+opacity: {
+  local/national/local/localStations: 0 to 1 // opacity for each layer, not the if layer is visible (which is set by view)
+}
+menu: true/false // display the top-right menu, default true
+shortFuseOnly: true/false // if only short-fused alerts (vs all hazards) should display (if visible), default false
+animating: true/false // if radar should enable play-back, default false
+basemap: 'standard', 'topographic', 'satallite', 'ocean', 'darkcanvas' // what basemap to display, default 'standard'
+artcc/cwa/county/state/rfc: true/false // if overlay should display, default false
+}
+*/
+
+const setupRadar = () => {
+  const container = document.querySelector("wx-radar");
+
+  const lat = Number.parseFloat(container.getAttribute("lat"));
+  const lon = Number.parseFloat(container.getAttribute("lon"));
+
+  const point = [lon, lat];
+
+  // no alerts;
+  // different heights on different breakpoints
+  // explore customizing the controls
+  // for a11y, describe that it's a radar map and otherwise hide it
+
+  const options = {
+    settings: {
+      menu: {
+        open: true,
+      },
+      map: {
+        base: "standard",
+        overlays: {
+          artcc: false,
+          county: false,
+          cwa: false,
+          rfc: false,
+          state: false,
+        },
+        animating: false,
+        zoom: 9,
+        center: point,
+        location: point,
+      },
+      agenda: null,
+      layers: {
+        alerts: {
+          filter: "hazards",
+          opacity: 0,
+        },
+        local: {
+          opacity: 0.6,
+        },
+        stations: {
+          opacity: 0.8,
+        },
+        national: {
+          opacity: 0.6,
+        },
+      },
+    },
+    urls: {
+      alerts: "https://alerts-v2.weather.gov",
+      api: "https://api.weather.gov",
+      forecast: "https://forecast.weather.gov",
+      gis: "https://opengeo.ncep.noaa.gov/geoserver",
+    },
+  };
+
+  const go = () => {
+    window.app = window.cmiRadar.createApp("#wx_radar_container", options);
+    window.app.$store.dispatch("markLocation", point);
+
+    [".cmi-radar-menu-container", ".cmi-radar-menu-agendas"].forEach(
+      (selector) => {
+        const element = document.querySelector(selector);
+        if (element) {
+          element.remove();
+        }
+      },
+    );
+  };
+
+  if (container.offsetParent) {
+    go();
+  } else {
+    const observer = new IntersectionObserver(
+      (entries, self) => {
+        if (entries[0].isIntersecting) {
+          go();
+          self.disconnect();
+        }
+      },
+      { root: document.body, rootMartin: "0px", threshold: 1 },
```

Comment:
I'm assuming this was just a typo:
```suggestion
      { root: document.body, rootMargin: "0px", threshold: 1 },
```